file-uploader-util.js

```diff
@@ -42,18 +42,18 @@ export function validateSubmission(submission, submissionFormat, baseOptions) {
     // cmsWebInterface is not an allowed submission method via file upload
     let errorDetails = [];
     validatedSubmission.measurementSets.forEach((ms, i) => {
-      if (ms.submissionMethod === 'cmsWebInterface') {
+      if (['cmsWebInterface', 'webAttestation'].indexOf(ms.submissionMethod) !== -1) {
         errorDetails.push([
           'submissionMethod',
           `measurementSets[${i}]`,
-          `'cmsWebInterface' is not allowed via file upload`
+          `'${ms.submissionMethod}' is not allowed via file upload`
         ]);
       }
     });
 
     if (errorDetails.length) {
       return Promise.reject(createErrorResponse('ValidationError',
-        `'cmsWebInterface' is not allowed via file upload`,
+        `Measurement set contains a disallowed submission method`,
```

Comment:
The previous message was more descriptive, should this new one be as well?

Reply:
@russbeye good point, but that message is a more generic message.  The `details` contains the detailed info.  For example, here's how the MA UI uses these:

![screen shot 2017-11-15 at 4 36 03 pm](https://user-images.githubusercontent.com/5840836/32861622-3b030360-ca23-11e7-91de-30477e40bc91.png)

![screen shot 2017-11-15 at 4 36 16 pm](https://user-images.githubusercontent.com/5840836/32861635-432e7d30-ca23-11e7-8600-bce5afc10abd.png)

Reply:
💥 Awesome!